workspaces/cicd-statistics/plugins/cicd-statistics/src/alpha/entityContent.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { convertLegacyRouteRef } from '@backstage/core-compat-api';
+import { EntityContentBlueprint } from '@backstage/plugin-catalog-react/alpha';
+import { rootCatalogCicdStatsRouteRef } from '../plugin';
+
+/**
+ * @alpha
+ */
+export const entityCicdChartsContent = EntityContentBlueprint.make({
+  name: 'entity',
+  params: {
```

Comment:
Should we have a filter for Kinds? Just thinking of some of the others we've done so it doesn't show up for every single Kind. Something like this:

https://github.com/backstage/community-plugins/blob/eb27d1bc29d64a7044a6481ead3474d4c30b6de5/workspaces/github-actions/plugins/github-actions/src/alpha/entityContent.tsx#L14